src/common/utxobased/db/makeProcessor.ts

```diff
@@ -793,7 +811,12 @@ const fetchAddressesByScriptPubkeys = async (
 
   // Short circuit query to the database
   if (scriptPubkeys.length === 0) return []
-  return await tables.addressByScriptPubkey.query('', scriptPubkeys)
+  const cleaner = asArray(asOptional(asIAddressCleaner))
+  return cleaner(
+    await tables.addressByScriptPubkey.query('', scriptPubkeys)
+    // allow any for cleaned string literal type
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  ) as any
```

Comment:
I'm not seeing the reason for `as any` type assertion here. Please explain.

Reply:
There is a comment in the line above explaining it - is it not clear enough? I have not been able to get cleaners to work with either enums or like in this case string literals in the time I spent with them so far.

Reply:
Here's an example on how to resolve the issue: https://github.com/EdgeApp/edge-currency-plugins/commit/97f0c0ae4b863af34649d73b3622d466b3f6bbb0 

I'll add something that I originally overlooked: all cleaners that have a separate static type definition should include that static type as the type variable/argument to the `asObject` cleaner in order to tightly couple the two definitions and receive errors from the type-checker if the two definitions misalign.

Reply:
I removed all the any type assertions in https://github.com/EdgeApp/edge-currency-plugins/pull/151/commits/c350510589e93b090a3f5bd819fa4015e685bc23 . Also implemented a custom cleaner for the enum type.